pycbc/types/timeseries.py

```diff
@@ -203,7 +203,7 @@ def at_time(self, time, nearest_sample=False):
         """
         if nearest_sample:
             time += self.delta_t / 2.0
-        return self[int((time-self.start_time)*self.sample_rate)]
+        return self[int(round((time-self.start_time)*self.sample_rate))]
```

Comment:
@bhooshan-gadre This should not be added. You should given the 'nearest_sample' option if you want this behavior which is already implemented.